packages/firestore/src/local/simple_db.ts

```diff
@@ -181,6 +191,25 @@ export class SimpleDb {
     return txn.store<KeyType, ValueType>(store);
   }
 
+  /** Parse User Agent to determine iOS version. Returns -1 if not found. */
+  static getIOSVersion(ua: string): number {
```

Comment:
Can you add a "// visible for testing" or similar comment to these methods to make it clear why they're public (and discourage dependencies from other code).